app/models.py

```diff
@@ -14,6 +14,36 @@
     JSONEncodedVARCHAR, MutableList
 
 
+class Permission(object):
+    """User permission definitions.
+
+    Attributes
+    ----------
+    ADMIN_USER
+        Permission to create a new API user, view API users, and modify
+        API user permissions.
+    ADMIN_PRODUCT
+        Permission to add, modify and deprecate Products.
+    ADMIN_EDITION
+        Permission to add, modify and deprecate Editions.
+    UPLOAD_BUILD
+        Permission to create a new Build.
+    DEPRECATE_BUILD
+        Permission to deprecate a Build.
+    """
+
+    ADMIN_USER = 0b1
+    ADMIN_PRODUCT = 0b10
+    ADMIN_EDITION = 0b100
+    UPLOAD_BUILD = 0b1000
+    DEPRECATE_BUILD = 0b10000
+
+    @classmethod
+    def full_permissions(self):
+        return self.ADMIN_USER | self.ADMIN_PRODUCT | self.ADMIN_EDITION \
+            | self.UPLOAD_BUILD | self.DEPRECATE_BUILD
```

Comment:
Shouldn't these be & together instead of |? Or is full permissions any permission?